doc/dune

```diff
@@ -0,0 +1,5 @@
+
+(mdx 
+  (packages current current_git current_docker)
```

Comment:
Probably also needs a `(package current_examples)` here, so it doesn't try to build it for the other packages. Might need to depend on dune 2.9 for that.